docs/user_documentation.rst

```diff
@@ -219,3 +291,46 @@ The GUI workflow is dead simple:
   is needed (such as confirming dependencies to install), DevAssistant will
   ask you, so don't go get your coffee just yet.
 - After all is done, get your coffee and enjoy.
+
+Where are the assistants located?
+---------------------------------
+
+You might ask, where are DAPs installed to? And you might already know the sort answer -
+to ``~/.devassistant``. However, the long answer is little bit more complicated.
+
+First of all, there are two things defined: ``DEVASSISTANT_HOME`` and ``DEVASSISTANT_PATH``.
+Be default, ``DEVASSISTANT_HOME`` is set to ``~/.devassistant`` and ``DEVASSISTANT_PATH`` contains
+the following:
+
+- ``~/.devassistant``
+- ``/usr/local/share/devassistant``
+- ``/usr/share/devassistant``
+
+All DAPs are installed to ``DEVASSISTANT_HOME``, however, you can run assistants from all the
+locations present in ``DEVASSISTANT_PATH``. If the assistant with the same name is present in more
+of them, locations on the beginning of the list have bigger priority.
+
+Sometimes the values of ``DEVASSISTANT_HOME`` and ``DEVASSISTANT_PATH`` are different. For example
+when installing DAPs as root, ``~/.devassistant`` is no longer there and
+``/usr/local/share/devassistant`` is used as ``DEVASSISTANT_HOME`` instead. So all the users can use
+assistants installed by root. You can also redefine the values entirely by using environment
+variables. The following example will install DAP ``python`` to ``.bah``::
+
+   $ DEVASSISTANT_HOME=.bah da pkg install python
+
+However you now cannot use assistants from ``python`` DAP unless you use that environment variable
+again.
```

Comment:
However, be advised that you cannot use the ``python`` assistants installed this way if you don't specify the ``DEVASSISTANT_HOME`` variable when running DevAssistant again.